README.md

```diff
@@ -40,7 +40,7 @@ environment variables (also found at `.env.example`):
 
```

Comment:
Just noticed I can't find the `.env.example` file. I think I forgot to comment on the last PR about that :)